src/templates/bootstrap/radio/form.ejs

```diff
@@ -1,6 +1,8 @@
 <div class="form-radio radio">
   {% ctx.values.forEach(function(item) { %}
-  <div class="form-check{{ctx.inline ? '-inline' : ''}}" ref="wrapper">
+  <div class="form-check{{ctx.inline ? '-inline' : ''}} {% if ((ctx.disabled && ctx.value) && (ctx.value === item.value || (typeof ctx.value === 'object' && ctx.value.hasOwnProperty(item.value) && ctx.value[item.value]))) { %}
```

Comment:
This much logic should be placed within the class and then passed into the template as a ctx variable.

Reply:
this is done